yaksh/bash_code_evaluator.py

```diff
@@ -27,7 +28,7 @@ def teardown(self):
             delete_files(self.files)
         super(BashCodeEvaluator, self).teardown()
 
-    def check_code(self, user_answer, file_paths, test_case):
+    def check_code(self, user_answer, file_paths, partial_grading, test_case, marks):
```

Comment:
check_code now returns 3 values, so it will be better if the Returns description is modified in the method docstring.